package.json

```diff
@@ -32,6 +32,7 @@
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^0.28.5",
+    "edx-bootstrap": "^0.2.1",
```

Comment:
I believe this new devDependency is required due to https://github.com/edx/studio-frontend/blob/36782a58ade9e115a1b3231fc45efd67ef4ce685/config/webpack.dev.config.js#L54, does that seem right?

Reply:
Oooh, good catch -- that should probably actually go in prod `dependencies`. I'm surprised we weren't getting errors about that!

Reply:
Oh yeah, I didn't notice it in `webpack.prod.config.js` the first time. Updating...

Reply:
Actually, it'd be good to switch versions for this package since Andy recently changed its name. It is now `"@edx/edx-bootstrap": "0.3.3"`

Reply:
Is that going to land in edx-platform soon as well?

https://github.com/edx/edx-platform/blob/master/package.json#L14

Reply:
It probably should. @andy-armstrong -- are we good to add the new `@edx/edx-bootstrap` dependency into platform?

Reply:
@arizzitano I made that change in my hackathon PR but that is on hold temporarily. If you guys want to go ahead and make that change that would be great, or I can do it.

Reply:
I'll add it to https://github.com/edx/edx-platform/pull/16111, thanks @andy-armstrong!